common/src/main/java/com/alibaba/nacos/common/utils/InetAddressValidator.java

```diff
@@ -37,8 +37,10 @@
     
     private static final int FIVE = 5;
     
+    private static final String IPV4_TUPLE = "(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9][0-9]|[0-9])";
+    
     private static final Pattern IPV4_PATTERN = Pattern
```

Comment:
Please do not modify this regular expression, IPUtil related please refer to https://github.com/alibaba/nacos/pull/5564/files modification

Reply:
in https://github.com/alibaba/nacos/pull/5564/files

This PR will not pass the following unit test case (in #5538)
```java
Assert.assertEquals("", IPUtil.getIPFromString(
                "jdbc:mysql://666.288.333.444:3306/nacos_config_test?characterEncoding=utf8&connectTimeout=1000"
                        + "&socketTimeout=3000&autoReconnect=true&useUnicode=true&useSSL=false&serverTimezone=UTC"));
```

Reply:
I want to know why we need to nest and maintain two kinds of IP verification logic?

Reply:
@SunJiFengPlus This class is an validator for IP (including V6 and V4), which come from other community `Dynatrace LLC`. The reason why we include it is we need a useful validator to check the IPv6 address.

So there are two kinds of Ip validator temp. Maybe we can refactor `IpUtil` to use this validator only.

Reply:
This is using IPV4_TUPLE matching result of tuple.
![image](https://user-images.githubusercontent.com/1195743/117748271-47602100-b242-11eb-921d-e18e3c1b707a.png)
The original expression does not have this problem, but it has another problem: it will match the whole string as IP. It cannot match the string containing IP.
So, first take the IP out of the string containing the IP, and then verify the IP
Or do you have a better way to solve this problem?